apps/launch/common-util/constants/rpcs.ts

```diff
@@ -1,5 +1,7 @@
 import { RPC_URLS } from 'libs/util-constants/src';
 
+export type SupportedChain = 1 | 100 | 137; // TODO: make it dynamic
```

Comment:
feel free to pull main, should be there https://github.com/valory-xyz/autonolas-frontend-mono/blob/main/apps/launch/common-util/constants/stakingContract.ts#L37

Reply:
ah I see. it's expected b/c I only deployed contracts on those three chains, but on prod we're supposed to have addresses for all 7 chains. hmm maybe we can add some dummy addresses for now for each remaining chain?